### ISSUE TITLE
Deleting an init.lua inside a folder breaks code sycning

### ISSUE THREAD

**Issue by jakebball** [rojo-rbx/rojo#435]
I found out an issue that occurs which causes rojo to not sync up any code however still shows it being conncected in the studio plugin. The steps I did to reproduce the issue starts with creating a folder in vscode, adding the init file, and then removing the init file directly and NOT the parent folder. After I do these actions rojo then breaks leading to the issue I stated above.

**Comment by triankl3** [comment]
I agree, I have noticed this behavior as well. What also tends to happen for me is when I disconnect from the server in the plugin, and then try to connect again, it just keeps on looping the loading animation. I have to restart the Rojo server for it to start actually working again.
